crates/ruma-identifiers-validation/src/space_child_order.rs

```diff
@@ -0,0 +1,24 @@
+use crate::Error;
+
+/// Validate the order of an [`m.space.child`].
```

Comment:
It might not be immediately obvious that this is talking about a field
```suggestion
/// Validate the `order` of an [`m.space.child`].
```